pkg/errors/errorMessage.go

```diff
@@ -140,6 +140,8 @@ var (
 	GALASA_ERROR_INVALID_OUTPUT_FORMAT          = NewMessageType("GAL1067E: Unsupported value '%s' for parameter --format. Supported values are: %s", 1067, STACK_TRACE_WANTED)
 	GALASA_ERROR_QUERY_RUNS_FAILED              = NewMessageType("GAL1068E: Could not query run results. Reason: '%s'", 1068, STACK_TRACE_WANTED)
 	GALASA_ERROR_LOG_FILE_IS_A_FOLDER           = NewMessageType("GAL1069E: Could not open log file for writing. '%s' is a directory, the --log parameter should refer to a file path (existing or not), or '-' (the console)", 1069, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_RETRIEVING_ARTIFACTS_FAILED    = NewMessageType("GAL1070E: Could not get run artifacts. Reason: '%s'", 1070, STACK_TRACE_WANTED)
+	GALASA_ERROR_DOWNLOADING_ARTIFACT_FAILED    = NewMessageType("GAL1071E: Could not download artifact '%s'. Reason: '%s'", 1071, STACK_TRACE_WANTED)
```

Comment:
I was expecting to see some error about the folder already existing, and the --force flag not being used...

Reply:
This error already exists as shown below - I didn't want to create a duplicate.

```
GALASA_ERROR_CANNOT_OVERWRITE_FILE = NewMessageType("GAL1036E: File '%s' exists. Use the --force flag to overwrite it.", 1036, STACK_TRACE_WANTED)
```